### ISSUE TITLE
ENM Sheep in Antlion's Clothing - Tuchulcha failed to sandpit

### ISSUE THREAD

**Issue by icariumkupo** [EdenServer/community#1242]
Tuchulcha attempted to use sandpit on a summoner pet that was released at the same time. Tuchulcha failed to use sandpit and therefore remained up. But at this point he is now invincible for the remainder of the fight. So we timed out.

![Icarium_2019 03 10_152141](https://user-images.githubusercontent.com/37199798/54091855-b253fb80-434a-11e9-8714-fce6207fcda6.png)

**Comment by Mikokiko** [comment]
I had something similar that happened, he did sandpit on my pet, but did not go underground, i did not try to release before. Sandpit seems to have missed.

I tried to kill both my char to see if this would allow me to attack him.

I tried both physical and magical BP , none worked.

After both character die, he went back underground, but he was still invincible.

I got up and tried again. 
![Kiko_2019 07 04_134929](https://user-images.githubusercontent.com/43910014/60682824-c7591b00-9e62-11e9-9fb1-a1a30f4e04ca.png)
I couldn't dmg it at all after that, I had to let the time run out.
